mkdocs.yml

```diff
@@ -32,5 +32,154 @@ extra_javascript:
   # Read the Docs requires JQuery for its JavaScript code to inject the flyout
   # menu. Material for MkDocs does not come with JQuery.
   - "https://code.jquery.com/jquery-3.6.2.min.js"
-#nav:
-#- Welcome: index.md
+nav:
+- Home: index.md
```

Comment:
This should be indented to match `- About CivicActions:`

Reply:
Fixed